docs/source/deployment/sql_server.rst

```diff
@@ -0,0 +1,212 @@
+
+    AS
+
+    BEGIN
+
+    EXEC sp_execute_external_script @language = N'Python',
+
+    @script = N'
+
+    from sklearn import datasets
+
+    iris = datasets.load_iris()
+
+    iris_data = pandas.DataFrame(iris.data)
+
+    iris_data["Species"] = pandas.Categorical.from_codes(iris.target, iris.target_names)
+
+    iris_data["SpeciesId"] = iris.target
+
+    ',
+
+    @input_data_1 = N'',
+
+    @output_data_1_name = N'iris_data'
+
+    WITH RESULT SETS (("Sepal.Length" float not null, "Sepal.Width" float not null, "Petal.Length" float not null, "Petal.Width" float not null,
+
+    "Species" varchar(100) not null, "SpeciesId" int not null));
+
+    END;
+
+    GO
+
+Finally insert data into the table iris_data and execute the procedure get_iris_dataset.
+
+.. code-block:: bash
+
+    INSERT INTO iris_data ("Sepal.Length", "Sepal.Width", "Petal.Length", "Petal.Width", "Species", "SpeciesId")
+
+    EXEC dbo.get_iris_dataset;
+
+
+
+The last step is to create a procedure for model deployment and prediction. We will create a procedure predict_species and as an external script we will run the bento saved bundle.
+
+.. code-block:: bash
+
+    import bentoml as usual and set the saved_path to the location where the bento bundle is saved.
+
+    Load the bundle using bentoml.load(). Now we can use this model loaded from the saved bundle to make predictions and deploy the model. List all the input and output features.
+
+    Here is the complete script
+
+    CREATE PROCEDURE predict_species (@model VARCHAR(100))
+
+    AS
+
+    BEGIN
+
+    `DECLARE @svm_model VARBINARY(max)`
+
+    `EXECUTE sp_execute_external_script @language = N'Python'`
+
+        `, @script = N'`
+    import bentoml
+
+    saved_path=r"C:\Program Files\Microsoft SQL Server\MSSQL15.NEWSERVER\bento_bundle"
+
+    irismodel = bentoml.load(saved_path)
+
+    species_pred = irismodel.predict(iris_data[["Sepal.Length", "Sepal.Width", "Petal.Length", "Petal.Width"]])
+
+    iris_data["PredictedSpecies"] = species_pred
+
+    OutputDataSet = iris_data[["id","SpeciesId","PredictedSpecies"]]
+
+    print(OutputDataSet)
+
+    ' , @input_data_1 = N'select id, "Sepal.Length", "Sepal.Width", "Petal.Length", "Petal.Width", "SpeciesId" from iris_data'
+
+        `, @input_data_1_name = N'iris_data'`
+        
+        `, @params = N'@svm_model varbinary(max)'`
+        
+        `, @nb_model = @svm_model`
+
+    `WITH RESULT SETS((`
+    
+                `"id" INT`
+            
+            `, "SpeciesId" INT`
+
+            `, "SpeciesId.Predicted" INT`
+            
+            ` ));`
+    END;
+
+    GO
+
+The procedure is ready now. We can deploy it using Execute predict_species 'SVM'; where SVM is the name given to the model.
+
+.. code-block:: bash
+
+    EXECUTE predict_species 'SVM';
+
+    GO
+
+
+After running the final query you can see the predictions in form of a table. Our model is served with SQL server easily with the help of BentoML.
```

Comment:
What are the steps that clean up this deployment on the SQL server?

Reply:
I believe overwriting or deleting the table will serve the purpose.  Users can create different prediction tables during the deployment of each model.  Do we need to mention this?

Reply:
If we can, I think we can mention how to delete the deployment. A user can run the full tutorial (from creating to testing to deleting) without worry about accumulating server cost.

Reply:
I did some research and found that as long as the remote computing is not connected no cost accumulation occurs. 
I am adding a one-line explanation for that and little information in the doc. Please check if that is relevant.